android/src/main/java/com/swmansion/reanimated/Utils.java

```diff
@@ -35,4 +35,15 @@ public static String simplifyStringNumbersList(String list) {
     // to make usage of std::istringstream in C++ easier
     return list.replace(",", "").replace("[", "").replace("]", "");
   }
+
+  public static float parseNumberToFloat(Object value) {
```

Comment:
This isn't parsing, rename this, e.g. `convertToFloat`

Reply:
Resolved in https://github.com/software-mansion/react-native-reanimated/pull/4103/commits/f0db7d894e6d402e8cb639f8e7fc52b108b65d95